### DISCUSSION TITLE
PictureController and hide default buttons

### DISCUSSION THREAD

**Discussion by arefrafei92** [Apparence-io/CamerAwesome#214] [General]
I used to take pictures by myself but I updated the library and PictureBuilder doesn't exist anymore.

**Comment by apalala-dev** [comment]
If you want to have your own UI for taking pictures, take a look at our documentation:
https://docs.page/Apparence-io/camera_awesome/getting_started/custom-ui
You may also be interested in other areas of the doc to see what you can do now since there have been many changes with version 1.0.0.
